apps/chromium-browser/docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
+  chromium:
```

I have also added the app proxy, so that we can ensure, that people log in first, before accessing the browser.
```suggestion
services:
  app_proxy:
    environment:
      APP_HOST: chromium-browser_chromium_1
      APP_PORT: 3000
     
  chromium:
```